src/Microsoft.IdentityModel.Tokens/CryptoProviderFactory.cs

```diff
@@ -189,13 +189,26 @@ public virtual KeyWrapProvider CreateKeyWrapProvider(SecurityKey key, string alg
             return CreateKeyWrapProvider(key, algorithm, false);
         }
 
-#pragma warning disable 1573
-        /// <inheritdoc cref="CreateForSigning(SecurityKey, string)" />
+        /// <summary>
+        /// Creates an instance of <see cref="KeyWrapProvider"/> for a specific &lt;SecurityKey, Algorithm>.
```

Comment:
Seems odd that there is both "&lt;" and ">" in the same line

Reply:
It's consistent with the rest of the library. '<' gets interpreted as an opening tag.